npm ERR! code ENOGIT no git found in your path

hi, please i need you help, i try to install the software but it crash and show this error:

alejandro@mesayuda:~/coopcycle-web$ docker-compose -f docker-compose.yml -f docker-compose-portainer.yml up

Creating volume "coopcycle-web_portainer_data" with default driver
Pulling smtp (namshi/smtp:)...
latest: Pulling from namshi/smtp
f189db1b88b3: Pull complete
eb1612d440dc: Pull complete
e65d3d397268: Pull complete
82d124bf3cdf: Pull complete
44f3200ac7ea: Pull complete
Building webpack
Step 1/7 : FROM node:8.12-alpine
 ---> df48b68da02a
Step 2/7 : WORKDIR /srv/coopcycle
 ---> Using cache
 ---> a36c3198b0e9
Step 3/7 : COPY package.json /srv/coopcycle
 ---> Using cache
 ---> bb049464dd37
Step 4/7 : COPY package-lock.json /srv/coopcycle
 ---> Using cache
 ---> ceb515da49f0
Step 5/7 : RUN npm install
 ---> Running in 0bdae2948184
npm ERR! code ENOGIT
npm ERR! Error while executing:
npm ERR! undefined ls-remote -h -t https://git@github.com/coopcycle/coopcycle-js.git
npm ERR! 
npm ERR! undefined
npm ERR! No git binary found in $PATH
npm ERR! 
npm ERR! Failed using git.
npm ERR! Please check if you have git installed and in your PATH.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-11-02T21_57_13_203Z-debug.log
ERROR: Service 'webpack' failed to build: The command '/bin/sh -c npm install' returned a non-zero code: 1

i have installed git and works fine, 

alejandro@mesayuda:~$ git --version
git version 2.17.1
alejandro@mesayuda:~$ type git
git está asociado (/usr/bin/git)
alejandro@mesayuda:~$ 

thanks for you help.

--- a/docker/webpack/Dockerfile
+++ b/docker/webpack/Dockerfile
@@ -4,6 +4,9 @@ WORKDIR /srv/coopcycle
 
 COPY package.json /srv/coopcycle
 COPY package-lock.json /srv/coopcycle
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh
 RUN npm install
 
 COPY docker/webpack/start.sh /

